app/views/users/edit.html.haml

```diff
@@ -14,6 +14,18 @@
         = link_to 'Change your avatar here', 'https://gravatar.com'
         = f.input :affiliation, as: :string,
           hint: 'This could be a company, a user group, or nothing at all.'
+        = f.input :website_url, type: :url, label: 'Personal Website'
+        = f.inputs :name => 'Social Media' do
+          = f.input :linkedin, type: :url
```

Comment:
I would like to see some hints for the different types. E.g. for the twitter handle I don't know if I have to enter ``@bruckmayer`` or ``https://twitter.com/bruckmayer``.